src/cephadm/cephadm.py

```diff
@@ -3187,7 +3193,6 @@ def get_container(ctx: CephadmContext,
         # So the container can modprobe iscsi_target_mod and have write perms
         # to configfs we need to make this a privileged container.
         privileged = True
-        set_pids_limit_unlimited(ctx, container_args)
```

Comment:
The comment above in this branch ("Applies only on rbd-target-api as get_tcmu_runner_container() removes all tcmu-runner arguments") is obsolete, right?  It looks like it should have been removed in https://github.com/ceph/ceph/pull/46506.

Reply:
yeah, seems like something that was added when setting the pids limit was being done differently earlier in the review process on the PR you linked. Now removing it as part of the commit here.